pages/inspire-books.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <link rel="stylesheet" href="../css/style.css">
+        <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+        <title>Inspire Books</title>
+    </head>
+    <div class="back-drop"></div>
+        <header>
+            <div class="navigation-background__color">
+                <div class="navigation-container">
+                    <div class="navigation-logo">
+                        <a href="#">
+                            <img src="../images/logo-light.png" height="60" alt="Inspire Academy logo white">
+                        </a>
+                    </div>
+                    <nav>
+                        <div class="hamburger-container">
+                            <div class="hamburger-button"><div class="before-after"></div></div>
+                        </div>
+                        <ul>
+                            <li><a href="#">INSPIRE BOOKS</a></li>
+                            <li><a href="#">22 CENTURY EDUCATION</a></li>
+                            <li><a href="#">TUTORING</a></li>
+                            <li><a href="#">SERVICES</a></li>
+                            <li><a href="#">ABOUT US</a></li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+        </header>
+        <!-- Mobile Navigation Bar -->
+        <div class="mobile-navigation">
+            <nav>
+                    <ul>
+                        <li><a href="#">INSPIRE BOOKS</a></li>
+                        <li><a href="#">22 CENTURY EDUCATION</a></li>
+                        <li><a href="#">TUTORING</a></li>
+                        <li><a href="#">SERVICES</a></li>
+                        <li><a href="#">ABOUT US</a></li>
+                    </ul>
+                </nav>
+        </div>
+        <main>
+            <!-- Inspire Books Logo - qod-dante -->
+            <img src="../images/inspire-books-logo.png" alt="Inspire Books Logo" width="500" height="500">
```

Comment:
remove the `width` attribute

Reply:
I will. I just did so because I read that not giving an image width or height will cause it to become unstable and cause errors. And it was also a temporary placeholder that the CSS "Generals" will remove and style properly in using CSS.

Reply:
Oh yes! I'm with you 100%
So instead of removing the width attributes, please set them to `auto`

like so width="auto" ;)